packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -309,16 +324,22 @@ export class RunningSummarizer implements IDisposable {
         })().finally(() => {
             summarizingLock.resolve();
             this.summarizingLock = undefined;
-            this.checkRerunHeuristics();
+            this.checkSummarizeAgain();
         }).catch((error) => {
             this.logger.sendErrorEvent({ eventName: "UnexpectedSummarizeError" }, error);
         });
     }
 
-    public summarizeOnDemand(
-        reason: string,
-        options: Omit<ISubmitSummaryOptions, "summaryLogger">,
-    ): OnDemandSummarizeResult {
+    public summarizeOnDemand({
```

Comment:
Consider adding an inheridoc comment here, similar to - https://github.com/microsoft/FluidFramework/blob/7f7a08a8e96babf4542583265ac46c5793bf21ec/packages/dds/shared-object-base/src/sharedObject.ts#L215

Reply:
For the function below too.